docs/UserGuide.md

```diff
@@ -91,7 +91,21 @@ Examples:
 
 Shows a list of all persons in the address book.
 
-Format: `list`
+Format: `list [s/FIELD]`
+
+* Sorts the contacts by the specified field in **ascending** order. `FIELD` must take one the following values:
+  * `d` Sort by date of birth
+  * `g` Sort by gender
+  * `n` Sort by number of products purchased
+  * `pa` Sort by purchase amount. i.e. Total amount ever spent
+
+* It is optional to specify the field to sort by. If no field is specified, persons are listed in the order they were inserted.
+* At most one field can be specified. i.e. Cannot specify 2nd or 3rd criteria to sort by.
+* Persons with an empty field that is being used to sort will be placed at the top of the list.
+
+Examples:
+* `list` Lists all persons without sorting them.
+* `list s/pa` Lists all persons sorted by their purchase amount in ascending order.
```

Comment:
Likewise, this line can be changed to something like:
`` `list s/g` Lists all persons sorted by their gender``